NSS-Enthusiastic-Camels/WorkforceManagement
# HR can view all departments including number of employees

# Link to Ticket
closes # Link

# Description of Proposed Changes
When Department Index is viewed you will see Department  Name,Budget and Number of Employees
## Steps to Test
1. `git fetch --all`
1. `git checkout gw-ticket6` 
1. Run BangazonWorkforce.
1. Browser opens.
1. Press Departments tab in navbar.
1. All departments are listed, the name, budget and number of employees are visible
1. In visual studio press the test tab.
1. Move mouse down to the Windows tab.
1. Press on Test Explorer.
1. in text explorer, right click on 'Get_IndexReturnsSuccessAndCorrectContentType'
1. press on 'run selected test'.


-see that a green check mark appears next to the test.

# Impacted Areas in Application
List general components of the application that this PR will affect:
DepartmentController.cs
DepartmentListViewModel.cs
Department Index.cshtml

Mentions @username
@stjepad 
@mateusvanhalen
@ThraxHamilton


Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each.
